content/tbd/1000 Landscapes for 1 billion people.md

```diff
@@ -0,0 +1,53 @@
+---
+created: 2023-11-30
+contributors:
+  - Catherine Tran
+title: 1000 Landscapes for 1 Billion People
+url: https://landscapes.global/
+locations:
+  - International network
```

Comment:
i don't think international network is their location? ...

Reply:
What happened here is: There's no one or few specific locations to fill in here as they are an international network. 

I think we should: 
- clarify that frontmatter fields should only contain data in the precise format specified in documentation, i.e. for locations that is [city], country. Otherwise it is too tempting to put in data in other formats when things don't quite fit .. noting this in https://github.com/life-itself/ecosystem/issues/150.

Reply:
I changed international network to not-applicable. see question about not-found and not-applicable in https://github.com/life-itself/ecosystem/issues/150